src/lib/srdb1/schema/dialog_in.xml

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE table PUBLIC "-//kamailio.org//DTD DBSchema V1.1//EN"
+  "http://kamailio.org/pub/kamailio/dbschema/dtd/1.1/dbschema.dtd" [
+
+<!ENTITY % entities SYSTEM "entities.xml">
+%entities;
+
+]>
+
+<table id="dialog_in" xmlns:db="http://docbook.org/ns/docbook">
+	<name>dialog_in</name>
+	<version>7</version>
+	<type db="mysql">&MYSQL_TABLE_TYPE;</type>
+	<description>
+		<db:para>Persistent dialog location information for the ims_dialog module.
+		More information can be found at:
+		<itemizedlist>
+		<listitem>
+		<ulink url="&KAMAILIO_MOD_DOC;ims_dialog.html">&KAMAILIO_MOD_DOC;ims_dialog.html</ulink>.
+		</listitem>
+		</itemizedlist>
+			</db:para>
+	</description>
+
+	<column id="id">
+		<name>id</name>
+		<type>unsigned int</type>
+		<size>&table_id_len;</size>
+		<autoincrement/>
+		<primary/>
+		<type db="dbtext">int,auto</type>
+		<description>unique ID</description>
+  </column>
+
+	<column id="hash_entry">
+		<name>hash_entry</name>
+		<type>unsigned int</type>
+		<size>10</size>
+		<description>Number of the hash entry in the dialog hash table
+		</description>
+		<natural/>
+	</column>
+
+	<column id="hash_id">
+		<name>hash_id</name>
+		<type>unsigned int</type>
+		<size>10</size>
+		<description>The ID on the hash entry</description>
+		<natural/>
+	</column>
+
+	<column>
+		<name>did</name>
+		<type>string</type>
+		<size>45</size>
+		<description></description>
+	</column>
+
+	<column>
+		<name>callid</name>
+		<type>string</type>
+		<size>&callid_len;</size>
+		<description>Call-ID of the dialog</description>
+	</column>
+
+	<column>
+		<name>from_uri</name>
+		<type>string</type>
+		<size>&uri_len;</size>
+		<description>The URI of the FROM header (as per INVITE)</description>
+	</column>
+
+	<column>
+		<name>from_tag</name>
+		<type>string</type>
+		<size>&from_tag_len;</size>
+		<description>The tag parameter serves as a general mechanism to
+		identify a dialog, which is the combination of the Call-ID along
+		with two tags, one from participant in the dialog.</description>
+	</column>
+
+	<column>
+		<name>caller_cseq</name>
```

Comment:
This column was called "caller_original_cseq" previously, and its also like this in the code. We can change it, but then we also need to adapt the module code I think.

Reply:
yes, correct. Fixed.